rascal-vscode-extension/package.json

```diff
@@ -14,7 +14,7 @@
     "color": "#ffffff",
     "theme": "light"
   },
-  "version": "0.13.0-head",
+  "version": "0.13.0-opendag",
```

Comment:
this shouldn't be committed. it's something you do locally. but stuff breaks if we merge this to main.

Reply:
thanks for the reminder